### PR TITLE
changed the default vmImage: ubuntu-latest to agent pools.

### PR DESCRIPTION
**Pull request by Makdaam**
WIP - untill DS2 VMs are deployed in ARO-CI and ARO-E2E pools (I'm running the pipelines to do that now).

I'm not sure if I should use the ARO-Build agent pool in .pipelines/build-and-push-images.yml (and also add nodes there - it's empty right now) or just use the ARO-CI pool for that and adjust the scale set size based on the job queue.

cc @jim-minter, @julienstroheker

### CODE REVIEW

File: .pipelines/deploy-dev-env.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ variables:
 jobs:
 - job: Deploy_new_shared_env
   pool:
-    vmImage: ubuntu-latest
+    name: ARO-CI
```
Comment (julienstroheker) [inline .pipelines/deploy-dev-env.yml:11]: ARO-Release no ?

Comment (Makdaam) [reply .pipelines/deploy-dev-env.yml]: indeed, changed to ARO-Release.

**Comment by mjudeikis** [comment]
@Makdaam how we can check if these will work? Does new Pool has the right configuration layered?

**Comment by Makdaam** [comment]
@mjudeikis You can check the results of the pipelines runs here https://msazure.visualstudio.com/AzureRedHatOpenShift/_settings/agentqueues also what nodes are deployed there. The job queue seems to be ok (either they're taken off the queue instantly or there are a few min of delay for ARO-CI, for ARO-E2E there are 30-40min wait times for some reason). 

Which parts are breaking CI?

**Comment by mjudeikis** [comment]
@jim-minter lets merge and deal with this after?

**Pull request merged**